packages/firestore/test/unit/specs/listen_spec.test.ts

```diff
@@ -785,4 +785,36 @@ describeSpec('Listens:', [], () => {
       .client(2)
       .expectEvents(query, { added: [docB] });
   });
+
+  specTest(
+    'Query is established in newly started client',
+    ['multi-client'],
+    () => {
+      const query = Query.atPath(path('collection'));
+      const docA = doc('collection/a', 1000, { key: 'a' });
+      const docB = doc('collection/b', 2000, { key: 'b' });
+
+      // Client 0 executes a query on behalf of Client 1. When client 0 shuts
+      // down, client 1 starts up and becomes primary, taking ownership of the
```

Comment:
I don't think this description is right.  You have a "client(2)" that starts up and becomes primary...